packages/fxa-content-server/app/scripts/lib/app-start.js

```diff
@@ -394,13 +375,14 @@ Start.prototype = {
     if (! this._user) {
       const user = this._user = new User({
         assertion: this._assertionLibrary,
+        config: this._config,
         fxaClient: this._fxaClient,
-        marketingEmailClient: this._marketingEmailClient,
+        //marketingEmailClient: this._marketingEmailClient,
```

Comment:
This should be removed right? cause it doesn't need to be passed in with options. @shane-tomlinson

Reply:
Yup! This reference should be removed.